.github/workflows/lint.yml

```diff
@@ -91,3 +91,12 @@ jobs:
     - name: lint remain
       run: |
         make lint-remain
+
+    - name: shellcheck
```

Comment:
Please name it in the pattern `lint xxx` and put it before `lint remain`.

Reply:
done, PTAL